lib/pages/settings_pages/components/settings_menu_widget.dart

```diff
@@ -4,27 +4,32 @@ import 'package:mosquito_alert_app/utils/style.dart';
 class SettingsMenuWidget extends StatelessWidget {
   final String? text;
   final Function onTap;
+  final bool addDivider;
 
-  SettingsMenuWidget(this.text, this.onTap);
+  SettingsMenuWidget(this.text, this.onTap, {this.addDivider = true});
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-          borderRadius: BorderRadius.circular(5),
-          color: Colors.white,
-          border: Border.all(color: Colors.black.withOpacity(0.1))),
-      child: ListTile(
-        onTap: () {
-          onTap();
-        },
-        title: Style.body(text),
-        trailing: Icon(
-          Icons.arrow_forward_ios,
-          color: Colors.black,
-          size: 18,
+    return Column(
+      children: [
+        ListTile(
+          onTap: () {
+            onTap();
+          },
+          title: Style.body(text),
+          trailing: Icon(
+            Icons.arrow_forward_ios,
+            color: Colors.black,
+            size: 18,
+          ),
+          visualDensity: VisualDensity(vertical: -4),
+          dense: true,          
         ),
-      ),
+        if (addDivider)
+          Container( padding: EdgeInsets.only(left: 10.0, right: 10.0),
```

Comment:
I was thinking if this padding could be replaced by a combination of `Divider.height`, `Divider.indent` and `Divider.endIndent` parameters